src/components/ui/chart.tsx

```
@@ -0,0 +1,369 @@
+"use client"
+
```

ESLint

[error] Missing semicolon.

(semi)